src/codecoverage/bot/code_coverage_bot/taskcluster.py

```diff
@@ -44,15 +47,13 @@ def get_task(branch, revision, platform):
 
 
 def get_task_details(task_id):
-    r = requests.get(queue_base + 'task/{}'.format(task_id))
-    r.raise_for_status()
-    return r.json()
+    task_details_json = queue_service.task(task_id)
+    return task_details_json
 
 
 def get_task_status(task_id):
-    r = requests.get(queue_base + 'task/{}/status'.format(task_id))
-    r.raise_for_status()
-    return r.json()
+    task_status_json = queue_service.task(task_id)
+    return task_status_json
```

Comment:
Are you sure this is correct? You made `get_task_status` exactly the same as `get_task_details`

Reply:
Now?

Reply:
Looks better, but the build is failing because you need to create the services passing client_id and access_token (see for example the notify service in notifier.py).

Also, since these two are just one liners now, you can just remove them and call those taskcluster-client functions directly where needed.

Reply:
Also, Does `get_task_artifacts` method return the artifacts for the latest run of the task?

Reply:
Didn't get you regarding: `Also, since these two are just one liners now, you can just remove them and call those taskcluster-client functions directly where needed.`

Reply:
`get_task_details` and `get_task_status` now only contain one line, so they are useless functions.
At this point, you can simply remove them. In the places where they are used, you can use the taskcluster-client functions directly.

Reply:
Ohh, cool